docs/tutorial/images.rst

```diff
@@ -12,73 +12,88 @@ inside of it, run::
     list(file.scans)  # e.g. shows: "['reference', 'bleach', 'imaging']"
 
-Once again, :attr:`.scans <lumicks.pylake.File.scans>` is a regular Python dictionary so we can
-easily iterate over it::
+:attr:`.scans <lumicks.pylake.File.scans>` is a regular Python dictionary so we can iterate over it::
 
     # Plot all scans in a file
     for name, scan in file.scans.items():
         scan.plot(channel="rgb")
         plt.savefig(name)
 
-Or just pick a single one::
+Or just pick a single one by providing the name of the scan as ``scan=file.scans["name"]``::
 
-    scan = file.scans["name"]
+    scan = file.scans["41"]
     scan.plot("red")
 
-Scan data and details
----------------------
+Plotting and Exporting
+----------------------
 
-You can access the raw image data directly. For a :class:`~lumicks.pylake.scan.Scan` with only a single frame::
+As shown above, there are convenience functions for plotting either the full RGB image or a single
+color channel::
 
-    rgb = scan.get_image("rgb")  # matrix with `shape == (h, w, 3)`
-    blue = scan.get_image("blue")  # single color so `shape == (h, w)`
+    scan.plot(channel="rgb")
 
-    # Plot manually
-    plt.imshow(rgb)
+The `channel` argument accepts the strings “red”, “green”, “blue”, or “rgb”. Multi-frame scans are also supported::
 
-For scans with multiple frames::
+    multiframe_file = lk.File("Scan_WTCas9_2Markers_binding on and off_stack.h5")
+    multiframe_scan = multiframe_file.scans["46"]
 
-    # returned data has `shape == (n_frames, h, w, 3)`
-    rgb = multiframe_scan.get_image("rgb")
-    # returned data has `shape == (n_frames, h, w)`
-    blue = multiframe_scan.get_image("blue")
+    print(multiframe_scan.num_frames)
+    print(multiframe_scan.get_image("blue").shape)  # (self.num_frames, h, w) -> single color channel
+    print(multiframe_scan.get_image("rgb").shape)  # (self.num_frames, h, w, 3) -> three color channels
 
-    # Manually plot the RGB image of the first frame.
-    plt.imshow(rgb[0, :, :, :])
+    # plot frame at index 3 (first frame is index 0)
+    # defaults to the first frame if no argument is given
+    multiframe_scan.plot("green", frame=3)
 
-We can also slice out a subset of frames from an image stack::
+Sometimes a few bright pixels can dominate the colormap of a scan.
+When this is the case, it may be beneficial to manually set the color limits for each of the channels.
+This can be accomplished by providing a :class:`~lumicks.pylake.ColorAdjustment` to plotting or export functions::
 
-    sliced_scan = multiframe_scan[5:10]
+    scan.plot(channel="rgb", adjustment=lk.ColorAdjustment([0, 0, 0], [4, 4, 4]))
 
-This will return a new :class:`~lumicks.pylake.scan.Scan` containing data equivalent to::
+Gamma adjustments can be applied in addition to the bounds by supplying an extra argument named `gamma`.
+For example, a gamma adjustment of `0.1` to the green channel can be applied as follows::
```

Comment:
Could be nice to add a few sample curves of what curve such a gamma adjustment has? Or alternatively, maybe even just make the `Gamma adjustments` text a link to wikipedia or another source so people can read about gamma adjustments there.

Reply:
How about https://en.wikipedia.org/wiki/Gamma_correction ?

Reply:
Yeah, that'd work :)

Reply:
I added the link, and also a bit more explanation on the color scale (that it is linear by default, but a gamma correction can be applied)